gips/data/core.py

```diff
@@ -701,8 +701,6 @@ def open_assets(self, product):
     # TODO - make general product_filter function
     def masks(self, patterns=None):
         """ List all products that are masks """
-        if patterns is None:
-            patterns = ['acca', 'fmask', 'mask']
         m = []
         for p in self.products:
             if any(pattern in p for pattern in patterns):
```

Comment:
If I try to iterate over `None` I get a TypeError:

```>>> [x for x in None]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable```

Does the signature of this method need to be changed to `def masks(self, patterns):`?